nixos/modules/services/misc/plex.nix

```diff
@@ -30,6 +30,30 @@ in
         default = "plex";
         description = "Group under which Plex runs.";
       };
+
+
+      managePlugins = mkOption {
```

Comment:
I think that old symlinks don't have to be cleared, because it's nix - every configuration there will be a new folder. So plugins will always be installed into empty `$PLUGINDIR`.